website/docs/security/authentication.md

```diff
@@ -103,3 +103,38 @@ If you encounter any issues while authenticating with FerretDB, try setting the
 Please refer to your specific driver documentation on how to set this field.
 
 If this does not resolve your issue please file a bug report [here](https://github.com/FerretDB/FerretDB/issues/new?assignees=ferretdb-bot&labels=code%2Fbug%2Cnot+ready&projects=&template=bug.yml).
+
+## Experimental authentication
+
+FerretDB supports experimental `SCRAM-SHA-256` and `SCRAM-SHA-1` authentication mechanisms.
+It is enabled by the flag `--test-enable-new-auth` or the environment variable `TestEnableNewAuth`.
+A user is created using the `createUser` command and stored in the `admin` database `system.users` collection.
```

Comment:
As discussed, we need a better introduction: what is it, what is the current status, what are upsides and downsides.

Environment variable name is wrong.